src/embed.js

```diff
@@ -81,9 +81,8 @@ function embed(el, spec, opt) {
     }
     mode = MODES[parsed.library];
 
-    parsedVersion = parsed.version.replace(/^v/g,'');
-    if (versionCompare(parsedVersion, VERSION[mode]) !== 0 ){
-      console.warn("The input spec uses \"" + mode + "\" " + parsedVersion + ", "
+    if (versionCompare(parsed.version, VERSION[mode]) !== 0 ){
```

Comment:
Yes, the warning is fine but what you wrote here means that the version has to be an exact match. I don't think that's necessary.